scripts/start.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+BASEDIR=$(dirname "$0")
+
+source ./$BASEDIR/.env
+
+# clear all previous docker artifacts
+./$BASEDIR/stop.sh
+
+# creating network to run all related containers in
+docker network create eschool-network
+
+# starting container with mysql DB
+docker run --name eschool-mysql \
+    -p 3606:3606 \
```

Comment:
Shouldn't it be -p 3606:3306?

Reply:
it is -p 3606:3606. Or i'm missing something?

Reply:
got it!

Reply:
fixed